Logger/LogFactory.cs

```diff
@@ -1,10 +1,17 @@
-﻿namespace Logger;
+namespace Logger;
 
 public class LogFactory
 {
+    private string? _filePath;
+
     public BaseLogger CreateLogger(string className)
     {
-        
-        return null;
+        if (_filePath is null) { return null!; }
+        return new FileLogger(_filePath, className);
```

Comment:
Instead of passing through the constructor, you should be using an object initializer 
```suggestion
        return new FileLogger(_filePath,)
        {
            ClassName = className
        }
```